test/regress/run_regression.py

```diff
@@ -313,15 +316,21 @@ def run_regression(unsat_cores, proofs, dump, use_skip_return_code, wrapper,
         return (EXIT_SKIP if use_skip_return_code else EXIT_OK)
 
     for req_feature in requires:
-        if req_feature.startswith("no-"):
-            inv_feature = req_feature[len("no-"):]
```

Comment:
Add `is_negative = True` to the if block and replace `inv_feature` with `req_feature`.